src/modules/referral/services/queries.ts

```diff
@@ -65,7 +69,11 @@ export const getTotalReferralRewardsQuery = () => {
         when d1."referralCount" >= 5 or d1."referralVolume" >= 100000 then 0.6
         when d1."referralCount" >= 3 or d1."referralVolume" >= 50000 then 0.5
         else 0.4
-      end as "referralRate"
+      end as "referralRate",
+      case
+          when d."depositDate"::date <= '2022-08-01' then 4
```

Comment:
Imagine after August 1 this date changes to be now September 1, will all rows including data from August 1 - September 1 be reset apppropraiately?